app/services/DR/requestManager.js

```diff
@@ -0,0 +1,183 @@
+  .then((raw) => data = raw.json())
+  .then(({ data }) => {
+    console.log('[INFO]',data);
+  })
+  .catch(() => {
+    console.log('[ERROR]');
+    return;
+  });
+
+  return await saveData(data, 'situation');
+}
+
+export async function requestBulletin() {
+  const url =
+    'http://digepisalud.gob.do/documentos/?drawer=Vigilancia%20Epidemiologica*Alertas%20epidemiologicas*Coronavirus*Nacional*Boletin%20Especial%20COVID-19';
+  // eslint-disable-next-line no-undef
+  const data = await fetch(url)
+    .then((res) => res.text())
+    .then((html) => {
+      const elems = html.match(/http:..digepisalud.gob.do.docs([^f]*.)/g);
+      const dates = html.match(/\d{1,2}\/\d{1,2}\/\d{4}([^am|pm]*)../g);
+      const urls = [...new Set(elems)];
+      const formatDate = (dateTime) => {
+        const [date, time, ampm] = dateTime.split(' ');
+        const [day, month, year] = date.split('/');
+        const months = [
+          'enero',
+          'febrero',
+          'marzo',
+          'abril',
+          'mayo',
+          'junio',
+          'julio',
+          'agosto',
+          'septiembre',
+          'octubre',
+          'noviembre',
+          'diciembre',
+        ];
+
+        return `${
+          months[month[1] - 1]
+        } ${day}, ${year} ${time} ${ampm}`.toUpperCase();
+      };
+
+      const data = urls.map((pdfUrl, i) => ({
+        title: `Boletín Especial ${i >= 9 ? i + 1 : `0${i + 1}`} COVID-19`,
+        date: formatDate(dates[i]),
+        image: {
+          source: require('../assets/images/bulletin.jpg'),
+        },
+
+        url: pdfUrl,
+      }));
+
+      return data.reverse();
+    })
+    .catch(console.error);
+  return await saveData(data, 'bulletins');
+}
+
+export function requestNews(pagination, callback) {
+  const section = pagination === 1 ? '' : `page/${pagination}/`;
+  const request = new Request(
+    `http://coronavirusrd.gob.do/category/noticias/${section}`
+  );
+
+  fetch(request).then((response) => {
+    if (response.status === 404) {
+      callback([], false);
+    }
+
+    response
+      .text()
+      .then((html) => {
+        env(html, (errors, { document }) => {
+          const titles = document.querySelectorAll('.title > h2');
+          const dates = document.querySelectorAll('.date');
+          const contents = document.querySelectorAll('.content');
+          const images = document.querySelectorAll('.photo > img');
+          const urls = document.querySelectorAll('.items a');
+          const res = [];
+          for (let i = 0; i < titles.length; i += 1) {
+            res.push({
+              title: titles[i].innerHTML,
+              date: dates[i].innerHTML,
+              content: contents[i].innerHTML,
+              image: {
+                source: require('../assets/images/new.jpg'),
+              },
+              url: urls[i].href,
+            });
+          }
+          callback(res, true);
+        });
+      })
+      .catch(console.error);
+  });
+}
```

Comment:
We don't need this, we are going to use the API for this

Reply:
done